docs/src/list_of_supported_models.md

```diff
@@ -34,9 +37,11 @@ independent assessment.
 [EvoTrees.jl](https://github.com/Evovest/EvoTrees.jl) | - | EvoTreeRegressor, EvoTreeClassifier, EvoTreeCount, EvoTreeGaussian, EvoTreeMLE | medium | tree-based gradient boosting models
 [EvoLinear.jl](https://github.com/jeremiedb/EvoLinear.jl) | - | EvoLinearRegressor | medium | linear boosting models
 [GLM.jl](https://github.com/JuliaStats/GLM.jl) | [MLJGLMInterface.jl](https://github.com/JuliaAI/MLJGLMInterface.jl) | LinearRegressor, LinearBinaryClassifier, LinearCountRegressor | medium² |
+[Imbalance.jl](https://github.com/JuliaAI/Imbalance.jl) | - | RandomOversampler, SMOTENC, TomekUndersampler, ClusterUndersampler, SMOTE, SMOTEN, ROSE, RandomUndersampler, ENNUndersampler, BorderlineSMOTE1, RandomWalkOversampler | low | 
```

Comment:
Assuming order matters here, it can be more intuitive to order them this way
```suggestion
[Imbalance.jl](https://github.com/JuliaAI/Imbalance.jl) | - | RandomOversampler, RandomWalkOversampler, ROSE, SMOTE, BorderlineSMOTE1, SMOTEN, SMOTENC, RandomUndersampler, ClusterUndersampler,  ENNUndersampler, TomekUndersampler, | low | 
```